picoscope/ps5000.py

```diff
@@ -264,13 +264,16 @@ def _lowLevelSetSimpleTrigger(self, enabled, trigsrc, threshold_adc,
         self.checkResult(m)
 
     def _lowLevelRunBlock(self, numPreTrigSamples, numPostTrigSamples,
-                          timebase, oversample, segmentIndex):
+                          timebase, oversample, segmentIndex, callback,
+                          pParameter):
         timeIndisposedMs = c_int32()
         m = self.lib.ps5000RunBlock(
             c_int16(self.handle), c_uint32(numPreTrigSamples),
             c_uint32(numPostTrigSamples), c_uint32(timebase),
             c_int16(oversample), byref(timeIndisposedMs),
             c_uint32(segmentIndex), c_void_p(), c_void_p())
+        # According to the documentation, 'callback, pParameter' should work
+        # instead of the last two c_void_p parameters.
```

Comment:
```suggestion
        # instead of the last two c_void_p parameters.
        # However to avoid the potential for serious crashes, we decided to
        # not include them in this function call.
```